### DISCUSSION TITLE
Account suspended

### DISCUSSION THREAD

**Discussion by brunosm97** [community/community#69134] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello, how are you?
My friend, who studies with me, had his account suspended. However, I believe there was a mistake because his account was new. He had to create this new account because the old one had been hacked. The mistake, I think, was that he used the same email as the old account. They suspended the wrong account; they were supposed to suspend the old one, but they suspended the new one, which had nothing wrong with it. It only had one repository, and that was the profile repository. He studies with me, and at our school, he cannot switch accounts to access the organization. He already sent an email six days ago, and they still haven't responded. He is missing course materials because he can't complete the activities.

The link to his new profile is: https://github.com/feroddev

And the account that was hacked is: https://github.com/neathfelipe

He doesn't know which email address the hacked account (neathfelipe) is linked to since the hackers removed his email from there. However, the new account is linked to the email: neath.felipe@gmail.com

If anyone can help, please assist him. He is already missing a lot of material, and the course is very challenging. Thanks.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @brunosm97, 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

**Reply by ghost** [reply]
Hi @queenofcorgis ,

My GitHub account: https://github.com/hv33y was suspended.

I don't know why as I didn't receive any mails or hints indicating why this happened.
I was taking to some one here in the community trying to solve a problem when suddenly it logged me out and when tried to sign in a message appear that account suspended for violating GitHub’s terms of service. Can some one please what is happening?
I am so sad and can't describe my sadness. I never violated TOS in my life and if that happened it is absolutely a mistake and not intentional. I’m happy to oblige with all of GitHub’s policies and Terms.

Thank you!

**Comment by Rana718** [comment]
My GitHub account (username: Luciferair) has been suspended without prior notification. I attempted to create a new support request on your Support website as instructed, but I encountered an issue and was unable to submit my request.

I urgently need assistance to resolve this problem and have my account reinstated. Could you please provide further instructions or assistance?
